Road/source/features/feature_selected.py

```diff
@@ -0,0 +1,66 @@
+# Standard library imports
+import ast
+import sys
+import os
+
+# Related third party imports
+import pandas as pd
+
+# Reset the path to the parent folder to import any mudule we want
+target = os.path.abspath(__file__)
+while (target.split("\\")[-1] != "Road"):
+    target = os.path.dirname(target)
+sys.path.append(target)
+
+# Local imports
+from source.constants import COLUMNS_TO_KEEP
+
+
+def delete_unnecessery_columns(data: pd.DataFrame) -> None:
+    """Remove all unnecessary columns except id, date, message,
+    from_id and reply_to columns.
+
+    Args:
+        data (pd.DataFrame): The dataframe that will be modify.
+
+    Returns:
+        None
+    """
+    return data[COLUMNS_TO_KEEP].copy()
+
+def extract_dates(date: pd.Series) -> None:
+    """Extract year, month, day, hour, minute, and second values
+    from the date series.
+
+    Args:
+        date (pd.Series): The date column to turn it into useful columns
+
+    Returns:
+        None
```

Comment:
Also here it returns the extracted year, month, day, etc. you can refer to them as a tuple of extracted features.

Reply:
Again, Forget to modify it after the refactoring.